UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Models/RadioNavigationalWarning/DTO/RadioNavigationalWarnings.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO
+{
+    public class RadioNavigationalWarnings
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        [DisplayName("Warning Type")]
+        [Required]
+        public int WarningType { get; set; }
```

Comment:
Shall we use Range attribute to validate WarningType has appropriate value?

Reply:
As the user selects from a fixed dropdown, I consider the range attribute is not required.

Reply:
Well that is an `assumption` you are making. One can post a body with any value and it is responsibility of validation routine to stop such requests. Kindly add a validation.

Reply:
thanks for highlighting, done required changes